geode-protobuf/src/main/java/org/apache/geode/protocol/protobuf/ProtobufOpsProcessor.java

```diff
@@ -44,9 +46,15 @@ public ProtobufOpsProcessor(SerializationService serializationService,
     ClientProtocol.Response.Builder builder;
     Result result;
     try {
-      result = operationContext.getOperationHandler().process(serializationService,
-          operationContext.getFromRequest().apply(request), context);
-    } catch (InvalidExecutionContextException e) {
+      if (context.getAuthorizer().authorize(operationContext.getAccessPermissionRequired())) {
+        result = operationContext.getOperationHandler().process(serializationService,
+            operationContext.getFromRequest().apply(request), context);
+      } else {
```

Comment:
We might want to log this. Logging all security related events can be a life saver during or after a security incident.